packages/js/src/components/social/SocialMetadata.js

```diff
@@ -21,17 +32,19 @@ const SocialMetadata = ( { displayFacebook, displayTwitter } ) => {
 			{ displayFacebook && <MetaboxCollapsible
 				hasSeparator={ false }
 				/* Translators: %s expands to Facebook. */
-				title={ sprintf( __( "%s preview", "wordpress-seo" ), "Facebook" ) }
+				title={ sprintf( __( "Social appearance", "wordpress-seo" ), "Facebook" ) }
```

Comment:
* If we use "Social appearance" as the translation string without using the `%s` variable, then I don't think we should use `sprintf` and passing `Facebook` as one of the arguments.
* I'd suggest to use:
   ````
   /* Translators: %s expands to Social. */
   title={ sprintf( __( "%s appearance", "wordpress-seo" ), "Social" ) }
   ```
OR
   ```
   title={ __( "Social appearance", "wordpress-seo" ) ) }
   ```